### PR TITLE
Improve hide toolbar when scrolling

### PR DESCRIPTION
**Pull request by RajaVamsi11**
I changed the code to make toolbar hide when user scroll up as
default.In setting it will give an option to enable the
"Keep Toolbar Visible".By enabling it toolbar will not hide as the
user scrolls up else default will work as usual.

Closes https://github.com/kiwix/kiwix-android/issues/10

### CODE REVIEW

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 1]
```diff
@@ -487,10 +490,18 @@ public void sectionsLoaded(String title, List<DocumentSection> sections) {
 
     View bookmarkTabView = LayoutInflater.from(KiwixMobileActivity.this)
             .inflate(R.layout.bookmark_tab, null);
-    bookmarkTabView.setOnClickListener(view -> PageBottomTab.of(4).select(pageActionTabsCallback));
-    bookmarkTabView.setOnLongClickListener(view -> {
-      PageBottomTab.of(4).longClick(pageActionTabsCallback);
-      return true;
+    bookmarkTabView.setOnClickListener(new View.OnClickListener() {
```
Comment (srv-twry) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Can you convert this to use lambda expressions ?

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 2]
```diff
@@ -487,10 +490,18 @@ public void sectionsLoaded(String title, List<DocumentSection> sections) {
...
+      @Override
+      public void onClick(View view) {
+        PageBottomTab.of(4).select(pageActionTabsCallback);
+      }
+    });
+    bookmarkTabView.setOnLongClickListener(new View.OnLongClickListener() {
```
Comment (srv-twry) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Same as above.

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 3]
```diff
@@ -613,21 +624,27 @@ private void setUpTTS() {
       @Override
       public void onSpeakingStarted() {
         isSpeaking = true;
-        runOnUiThread(() -> {
-          menu.findItem(R.id.menu_read_aloud)
-              .setTitle(createMenuItem(getResources().getString(R.string.menu_read_aloud_stop)));
-          TTSControls.setVisibility(View.VISIBLE);
+        runOnUiThread(new Runnable() {
```
Comment (srv-twry) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Same here.

Comment (RajaVamsi11) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: ok..Thank you for your reply

**Comment by RajaVamsi11** [comment]
@mhutti1  i have made the requested changes.

**Comment by RajaVamsi11** [comment]
@mhutti1 Take a look at it.

**Pull request merged**